resources/function-controller/charts/docker-registry/values.yaml

```diff
@@ -133,13 +133,25 @@ nodeSelector: {}
 
 tolerations: []
 
-extraVolumeMounts: []
+extraVolumeMounts:
+  - name: htpasswd-data
+    mountPath: /data
 ## Additional volumeMounts to the registry container.
 #  - mountPath: /secret-data
 #    name: cloudfront-pem-secret
 #    readOnly: true
 
-extraVolumes: []
+extraVolumes:
```

Comment:
Please override those values in root dir of `function-controller` chart. Please don't touch any files in not our charts.

Reply:
done